index.html

```diff
@@ -128,15 +128,15 @@ <h2>Tonic</h2>
             <div class="card-block" id="arti-2">
                 <h2>Multi-post Stories</h2>
                 <ul class="card-frame">
-                    <li class="no-count">Canopy</li>
-                    <li>Full Stack Dev</li>
+                    <li class="no-count">CANOPY</li>
+                    <li>Back End Dev</li>
                     <li>2015</li>
                 </ul>
-                <p>Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.</p>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
                 <ul class="card-tags">
                     <li>html</li>
+                    <li>CSS</li>
                     <li>Javascript</li>
-                    <li>css</li>
                 </ul>
                 <button class="card-button" type="submit">See Project</button>
       </div>
```

Comment:
- [ ] Your cards are looking great @brainskev, But there are something's to improve. Make sure to use the right font type and size. 

## figma design 
![figma](https://user-images.githubusercontent.com/69077061/198847010-24ed8d2b-0204-48f7-b541-5e06ea40266d.PNG)

## your design 
![your](https://user-images.githubusercontent.com/69077061/198847018-07da9e28-ec69-48da-a554-63f4bcc4e037.PNG)